pow/src/main/java/tech/pegasys/teku/pow/Eth1Provider.java

```diff
@@ -31,7 +28,7 @@
 
   SafeFuture<Block> getGuaranteedEth1BlockFuture(UnsignedLong blockNumber);
 
-  SafeFuture<Block> getLatestEth1BlockFuture();
+  SafeFuture<Block> getLatestEth1Block();
```

Comment:
nit: to keep the method names in line, you might want to either add Future to the end of this method name or remove all the Futures from the method names above.

Reply:
I had meant to remove future from them all but forgot to do a full pass will fix.